### PR TITLE
Fix paths for newest webgme-engine compatibility

### PR DESCRIPTION
**Pull request by ksmyth**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,16 +84,47 @@ <h1 class="hostname"><span></span> <small></small></h1>\
                             "executor": "common/executor"
                             // "jszip": 'lib/jszip/jszip',
                         },
+                        map: {
+                            '*': {
```
Comment (pmeijer) [inline index.html]: Are these mappings really needed?

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "webgme-executor-worker",
-  "version": "1.0.1",
+  "version": "2.0.0",
```
Comment (pmeijer) [inline package.json]: Please don't bump the version in the branch - it'll be done when making a release from master.

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   "description": "Worker for connecting to webgme executor framework",
-  "main": "node_worker.js",
+  "main": "index.html",
```
Comment (pmeijer) [inline package.json]: Why change this?

File: index.html (hunk 1 above)
Comment (ksmyth) [reply index.html]: No, I will remove them

File: package.json (hunk 2 above)
Comment (ksmyth) [reply package.json]: ok

File: package.json (hunk 3 above)
Comment (ksmyth) [reply package.json]: nw.js reads this field. I reverted my change back to node_worker.js so npm works

**Comment by ksmyth** [comment]
I pushed a new commit, rebased on current master. It addresses your comments and makes `npm test` work. Thanks

**Pull request merged**